src/goto-programs/cfg.h

```diff
@@ -399,12 +446,13 @@ void cfg_baset<T, P, I>::compute_edges(
     break;
 
   case ASSUME:
+  case ASSERT:
```

Comment:
I disagree with this change because I don't think that's the right semantics for assert.

Reply:
In general "you can't get past 'assert(false)'" seems reasonable, but I note in `--cover` mode we intentionally sprinkle `ASSERT(FALSE)`  all over the place and then selectively lift the assertions to try to achieve further coverage. In this case stopping at the first one isn't correct -- but I don't know if `cfgt` would ever be used on the GOTO program that had the coverage assertions added.